examples/resources/security_policy/resource_cluster_baseline_security_policy.tf

```diff
@@ -0,0 +1,29 @@
+# Create Tanzu Mission Control security policy
+resource "tanzu-mission-control_security_policy" "create_cluster_baseline_security_policy" {
```

Comment:
can you change "create_cluster_baseline_security_policy" to "cluster_scoped_baseline_security_policy". 
And do the similar changes to others as well/ 
Create does not go well, because the same resource is used for update as well. so create does not fit in

Reply:
Sure

Reply:
Done